code/controllers/subsystem/air.dm

```diff
@@ -1,11 +1,17 @@
+#define SSAIR_PIPENETS 1
+#define SSAIR_ATMOSMACHINERY 2
+#define SSAIR_ACTIVETURFS 3
+#define SSAIR_EXCITEDGROUPS 4
+#define SSAIR_HIGHPRESSURE 5
+#define SSAIR_HOTSPOTS 6
+#define SSAIR_SUPERCONDUCTIVITY 7
 var/datum/subsystem/air/SSair
 
 /datum/subsystem/air
 	name = "Air"
 	priority = -1
-	wait = 5
+	wait = 2.5
```

Comment:
Should I keep this?

Reminder that when atmos (or any dynamic wait subsystem) sleeps because it used up too much of a tick, it waits SS.wait before resuming.